process-engine/bpmn-studio
# 🚧 WIP: 💄 Fix Minor Style Bugs

## What did you change?

This PR fixes the following style bugs:

- Unintentional resizing of left toolbar
- Icons in left toolbar are not centered
- ConnectedTo Element is no status-bar__element
- status-bar__elements wrap when they have not enough space
- Login dropdown is no menu__element
- menu___elements wrap when they have not enough space
- diagram-tools-right are not scrollable
- The Process Details Dropdown Menu Entries have no spacing


## How can others test the changes?

- Open up the BPMN-Studio
- Check if the style bugs are gone

## PR-Checklist

Please check the boxes in this list after submitting your PR:
